packages/ag-charts-community/src/chart/toolbar/toolbar.ts

```diff
@@ -222,6 +226,21 @@ export class Toolbar extends BaseModuleInstance implements ModuleInstance {
         this.translateFloatingElements(FloatingTop, false);
     }
 
+    // AG-12695 Temporarily set `pointer-events: none` on the annotationOptions when dragging, because the
+    // buttons block to mouse from hovering over the canvas.
+    private readonly oldPointerEvents: Map<HTMLButtonElement, string> = new Map();
+    private onStopPointerEvents() {
+        if (this.oldPointerEvents.size !== 0) throw new Error('AG Charts - unexpected drag-start event');
+        this.groupButtons['annotationOptions'].forEach((b) => {
+            this.oldPointerEvents.set(b, b.style.pointerEvents);
```

Comment:
I think it would be better not to keep this state around, could we perhaps instead use a CSS class for this specific case, and add/remove it to apply/remove the `pointer-events: none`? This would be less likely to conflict with other processing (which is presumably why we're defensively copying the original `pointerEvents`)?

Reply:
What is the advantage of toggling a class over this approach? The advantage of setting style values on the element directly that these have priority over class values. This tells the browser ignore the class styling, which is what is intended here.